docs/en-US/component/mention.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Mention
+lang: en-US
+---
+
+# Mention
+
+Used to mention someone or something in an input.
+
+## Basic Usage
+
+The most basic usage.
+
+:::demo
+
+mention/basic
+
+:::
+
+## Textarea
+
+The input type can be set to `textarea`.
+
+:::demo
+
+mention/textarea
+
+:::
+
+## Customize label
+
+Customize label by `label` slot.
+
+:::demo
+
+mention/label
+
+:::
+
+## Load remote options
+
+Load options asynchronously.
+
+:::demo
+
+mention/loading
+
+:::
+
+## Customize trigger token
+
+Customize trigger token by `prefix` props. Default to `@`, `Array<string>` also supported.
+
+:::demo
+
+mention/prefix
+
+:::
+
+## Delete as a whole
+
+Set the `whole` attribute to `true`, and when you press the backspace, the mention will be deleted as a whole.
+Set the `check-is-whole` attribute to customize the checking logic.
+:::demo
+
+mention/whole
+
+:::
+
+## Work with form
+
+to work with `el-form`.
+
+:::demo
+
+mention/form
+
+:::
+
+## API
+
+### Attributes
+
+::: tip
+Since this component is developed based on the component [`el-input`](./input.md#attributes) , the original properties have not changed, so no repetition here,
+and please go to the original component to view the documentation.
+:::
```

Comment:
```suggestion
::: tip
Since this component is developed based on the component [`el-input`](./input.md#attributes) , the original properties have not changed, so no repetition here,
and please go to the original component to view the documentation.
:::

## API

### Attributes
```